.test-infra/dataproc/init-actions/flink.sh

```diff
@@ -121,7 +123,17 @@ function configure_flink() {
     grep 'spark\.executor\.cores' /etc/spark/conf/spark-defaults.conf \
       | tail -n1 \
       | cut -d'=' -f2)
-  local flink_taskmanager_slots="$(($spark_executor_cores * 2))"
```

Comment:
Could we duplicate the spark code here? Relying on an external config file is a bit strange.

Reply:
Not sure - this is taken from the original init action - I will investigate that too.

Reply:
I "paraphrased" the default behavior of Dataproc while it was generating the spark-default.conf file. LMK if it's not too cryptic. 

Another approach we could have here is to simply set 1 slot by default and provide slot number through metadata otherwise. Not cryptic and explicit. 

Which approach feels better to you?